demos/blockfactory/code_to_block_xml.js

```diff
@@ -0,0 +1,393 @@
+// Copyright 2017 Juan Carlos Orozco Arena
+// Apache License Version 2.0
+
+BlockFactory.manualEdit = function(){

```

Comment:
Please add [JSDoc](http://usejsdoc.org/) comments for each function, and a `/** @fileoverview ... */` comment for each new file.

Note that [Google's JavaScript style](https://google.github.io/styleguide/jsguide.html) requires a space between the `()` and `{` (here and many other places).